filebeat/filebeat.full.yml

```diff
@@ -209,7 +209,7 @@ filebeat.prospectors:
 
 # Name of the registry file. If a relative path is used, it is considered relative to the
 # data path.
-#filebeat.registry_file: registry
+#filebeat.registry_file: ${path.data}/registry
```

Comment:
Writing `filebeat.registry_file: ${path.data}/registry` will be equivalent with writing `filebeat.registry_file: registry`, right?

I like the version with `${path.data}` as it's more explicit about where the file is searched. But we'll need to explain in the docs that these variables are automatically set somehow.